modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -823,90 +823,192 @@ The JSON object is of the form:
 
 ## 5.0 DICOM Data
 
-Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
-portion of the REST API namespace, but is defined in a separate section of this 
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$VisitLabel`
```

Comment:
I just realized my above comments were made on the 0.0.3 documentation which shouldn't be changed because it's already released. Pretend I made them on the v0.0.4-dev..

Reply:
Isn't it ok to change the 0.0.3 documentation if it only clarifies the current behavior without changing it ? I would think it is ok personally.

Reply:
Yes, clarifying the documentation is fine, changing the behaviour (incl adding new things) should only go into the dev version. An API client that uses version $x of the API should be able to expect the same behaviour if they point to a different LORIS instance.

Reply:
So I am not sure of what you are asking me to do. Do you want the API DICOM upload endpoints to only be added to 0.0.4-dev ?

Reply:
Yes. (And pretend that the comments I made on them in 0.0.3 before realizing I was looking at the wrong one were made on 0.0.4-dev)

Reply:
Do you want me to duplicate the comments?

Reply:
No need to duplicate the comments, I am going through each of them one by one anyway.

Reply:
https://github.com/aces/Loris/commit/8230e890555a37af5a6301f4240e43172988c1a6

This commit removes the documentation from the API v0.0.3. In practice, the new endpoints were already v0.0.4-only, but there are a few v0.0.4 POST routes that are added to v0.0.3 endpoints, not sure what do do here (ignore or implement method specific supported versions ?).